### PR TITLE
chore: Standardize CI with other projects

### PR DESCRIPTION
**Pull request by ghislainbourgeois**

### CODE REVIEW

**Comment by ghislainbourgeois** [comment]
@gab-arrobo , this one also seems ignored by the onf-bot

**Comment by gab-arrobo** [comment]
@ghislainbourgeois @thakurajayL, I see this repo has only one file that might be relevant (path.go + its test file). What about moving the few things that are important from this repo into the `util` repo? So, we do not have to maintain multiple tiny repos for no reason. What do you guys think?

**Comment by ghislainbourgeois** [comment]
> @ghislainbourgeois @thakurajayL, I see this repo has only one file that might be relevant (path.go + its test file). What about moving the few things that are important from this repo into the `util` repo? So, we do not have to maintain multiple tiny repos for no reason. What do you guys think?

I think it would make sense, it would involve fixing the imports in other projects, but that should not take too long.

**Comment by gab-arrobo** [comment]
@ghislainbourgeois, should we close this PR? or should we merge it?

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "daily"
+
```
Comment (gab-arrobo) [inline .github/dependabot.yml:17]:
```suggestion
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: 'go.mod'
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v3.7.0
```
Comment (gab-arrobo) [inline .github/workflows/main.yml:37]:
```suggestion
        uses: golangci/golangci-lint-action@v4.0.0
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+  fossa-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: FOSSA scan
+        uses: fossa-contrib/fossa-action@v3
+        with:
+          fossa-api-key: 9dc8fa92e3dd565687317beb87b56d89
```
Comment (gab-arrobo) [inline .github/workflows/main.yml:58]:
```suggestion
          fossa-api-key: 0c3bbcdf20e157bbd487dae173751b28
```

**Review by gab-arrobo — CHANGES_REQUESTED** [review]
The `license-check` action fails because several files do not have copyright and license headers

**Comment by gab-arrobo** [comment]
@ghislainbourgeois, I am closing this PR given that it is not needed anymore. All of the repos are pointing to `path_util` from `util` repo

**Pull request closed without merging**